docs/kiali/assets/architecture.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The diagram needs to be pimped up to be compliant with our standards, please talk to @bszwarc